app/_components/TestComponent/BadComponent.tsx

```
@@ -0,0 +1,93 @@
+                <p>Count is: {this.state.count}</p>
+                <button onClick={() => this.setState({ count: this.state.count + 1 })}>
+                    Increase Count
+                </button>
+            </div>
+        );
+    }
+}
+
+const BadComponent = () => {
+    let [counter, setCounter] = useState(0);
+    let [dataList, setDataList] = useState(data);
```

ESLint

[error] 'setDataList' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)